### PR TITLE
Added support for tunnel route status in show vnet routes all.

### PR DESCRIPTION
**Pull request by siqbal1986**
#### What I did
This commit adds a new status filed in CLI "show vnet routes all"
#### How I did it
The status information is retrieved using the status of the prefix from Application DB.
The status field can have 3 values.
Empty : Configured but not in Application DB.
Active : Configured and active in ASIC.
Inacive : Configured but not in ASIC due to lack of BFD session with endpoint
#### How to verify it
Add a vxlan tunnel route with single Endpoint monitoring. run the command to see active/ inactive based on BFD session state.
if multiple Endpoints are present then atleast one has to be Up for status to show up as active.
if endpoint monitoring is not enabled then status would show up as active.
#### Previous command output (if the output of a command-line utility has changed)
vnet name        prefix                    endpoint                                     mac address    vni 
---------------  ------------------------  -------------------------------------------  -------------  -----
Vnet1            10.2.1.0/24               200.3.152.32                                                       
Vnet_v4_in_v4-0  150.62.191.1/32           100.251.99.1,100.251.99.2,100.251.99.3
Vnet_v6_in_v6-0  fddd:a150:a251::a6:1/128  fddd:a100:a251::a10:1,fddd:a101:a251::a10:1
test_v4_in_v4-0  160.62.191.1/32           100.251.7.1                                                        
test_v4_in_v4-0  160.63.191.1/32           100.251.7.2
test_v4_in_v4-0  160.64.191.1/32           100.251.7.3
#### New command output (if the output of a command-line utility has changed)
vnet name        prefix                    endpoint                                     mac address    vni    status
---------------  ------------------------  -------------------------------------------  -------------  -----  --------
Vnet1            10.2.1.0/24               200.3.152.32                                                       active
Vnet_v4_in_v4-0  150.62.191.1/32           100.251.99.1,100.251.99.2,100.251.99.3
Vnet_v6_in_v6-0  fddd:a150:a251::a6:1/128  fddd:a100:a251::a10:1,fddd:a101:a251::a10:1                        inactive
test_v4_in_v4-0  160.62.191.1/32           100.251.7.1                                                        inactive
test_v4_in_v4-0  160.63.191.1/32           100.251.7.2                                                        inactive
test_v4_in_v4-0  160.64.191.1/32           100.251.7.3                                                        inactive

### CODE REVIEW

File: show/vnet.py [hunk 1]
```diff
@@ -227,7 +228,7 @@ def all():
 
     click.echo()
 
-    header = ['vnet name', 'prefix', 'endpoint', 'mac address', 'vni']
+    header = ['vnet name', 'prefix', 'endpoint', 'mac address', 'vni', 'status']
```
Comment (prsunny) [inline show/vnet.py:231]: Please add tests so as to pass coverage

File: show/vnet.py [hunk 2]
```diff
@@ -237,10 +238,13 @@ def all():
     for k in vnet_rt_keys:
         r = []
         r.extend(k.split(":", 2)[1:])
+        state_db_key = '|'.join(k.split(":",2))
         val = appl_db.get_all(appl_db.APPL_DB, k)
+        val_state = state_db.get_all(state_db.STATE_DB, state_db_key)
         r.append(val.get('endpoint'))
         r.append(val.get('mac_address'))
         r.append(val.get('vni'))
+        r.append(val_state.get('state'))
```
Comment (prsunny) [inline show/vnet.py:247]: Can you check the output if the entry is not present in state_db?

File: show/vnet.py (hunk 1 above)
Comment (siqbal1986) [reply show/vnet.py:231]: done

File: show/vnet.py (hunk 2 above)
Comment (siqbal1986) [reply show/vnet.py:247]: done

**Review by prsunny — APPROVED** [review]

**Comment by prsunny** [comment]
Please raise a PR to 202012

**Review by zjswhhh — APPROVED** [review]

**Comment by yxieca** [comment]
@siqbal1986 , this change cannot be cherry-picked to 202205 branch cleanly. Can you raise a separate PR?

**Pull request merged**